estate/data/estate_property.xml

```diff
@@ -0,0 +1,191 @@
+        <field name="name">x_living_area</field>
+        <field name="field_description">Living Area</field>
+        <field name="ttype">integer</field>
+    </record>
+
+    <record id="field_estate_property_facades" model="ir.model.fields">
+        <field name="model_id" ref="model_estate_property" />
+        <field name="name">x_facades</field>
+        <field name="field_description">Facades</field>
+        <field name="ttype">integer</field>
+    </record>
+
+    <record id="field_estate_property_garage" model="ir.model.fields">
+        <field name="model_id" ref="model_estate_property" />
+        <field name="name">x_garage</field>
+        <field name="field_description">Garage</field>
+        <field name="ttype">boolean</field>
+    </record>
+
+    <record id="field_estate_property_garden" model="ir.model.fields">
+        <field name="model_id" ref="model_estate_property" />
+        <field name="name">x_garden</field>
+        <field name="field_description">Garden</field>
+        <field name="ttype">boolean</field>
+    </record>
+
+    <record id="field_estate_property_garden_area" model="ir.model.fields">
+        <field name="model_id" ref="model_estate_property" />
+        <field name="name">x_garden_area</field>
+        <field name="field_description">Garden Area</field>
+        <field name="ttype">integer</field>
+    </record>
+
+    <record id="field_estate_property_garden_orientation" model="ir.model.fields">
+        <field name="model_id" ref="model_estate_property" />
+        <field name="name">x_garden_orientation</field>
+        <field name="field_description">Garden Orientation</field>
+        <field name="ttype">selection</field>
+    </record>
+
+    <record id="field_estate_property_garden_orientation_north" model="ir.model.fields.selection">
+        <field name="field_id" ref="field_estate_property_garden_orientation" />
+        <field name="name">North</field>
+        <field name="value">north</field>
+    </record>
+
+    <record id="field_estate_property_garden_orientation_south" model="ir.model.fields.selection">
+        <field name="field_id" ref="field_estate_property_garden_orientation" />
+        <field name="name">South</field>
+        <field name="value">south</field>
+    </record>
+
+    <record id="field_estate_property_garden_orientation_east" model="ir.model.fields.selection">
+        <field name="field_id" ref="field_estate_property_garden_orientation" />
+        <field name="name">East</field>
+        <field name="value">east</field>
+    </record>
+
+    <record id="field_estate_property_garden_orientation_west" model="ir.model.fields.selection">
+        <field name="field_id" ref="field_estate_property_garden_orientation" />
+        <field name="name">West</field>
+        <field name="value">west</field>
+    </record>
+
+    <record id="field_estate_property_proprty_type_id" model="ir.model.fields">
+        <field name="model_id" ref="estate.model_estate_property" />
```

Comment:
`ref` to an id of the same module works without having to specify the complete xmlid
```suggestion
        <field name="model_id" ref="model_estate_property" />
```
This is not true when dealing with `env.ref`